cookbooks/van-keken-vof/doc/amr.part.prm

```diff
@@ -0,0 +1,8 @@
+subsection Mesh refinement
+  set Initial adaptive refinement        = 0
+  set Strategy                           = volume of fluid interface
+  set Initial global refinement          = 7
```

Comment:
That's quite a fine mesh. How long does the cookbook run? We try to keep the cookbooks so that they run in a reasonable amount of time by default, and if the mesh is too coarse to be accurate, then comment on that in the cookbook and possibly show results from a more refined mesh.

Reply:
@bangerth I believe we always ran the same meshes that Scott King, et al. ran for compositional fields, never finer.  VOF should always be less expensive than compositional fields on a problem with a well defined interface like this one.

Reply:
With 4 processors on Peloton2, the run took 1.15 hrs. I could adjust the version in the cookbook to 6 levels of refinement which took ~8 minutes if a 1hr runtime is too long.

Reply:
Yes, let's do that. We try to set the defaults in the cookbooks in such a way that things can be run on a laptop with reasonable effort so that people can immediately start to play.